2014/12/32014/12/03 15:33:46 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：主键异常！  
file:D:\wwwroot\veecar\framework\core\Model.php  
line:34  
function:__construct  
class:Ivy\core\Model  
type:->  
args:array (
  0 => NULL,
)  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:model  
class:Ivy\core\Model  
type:::  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Model.php on line 22

2014/12/03 15:33:46 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：主键异常！  
file:D:\wwwroot\veecar\framework\core\Model.php  
line:34  
function:__construct  
class:Ivy\core\Model  
type:->  
args:array (
  0 => NULL,
)  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:model  
class:Ivy\core\Model  
type:::  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Model.php on line 22

2014/12/03 15:35:02 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：主键异常！  
file:D:\wwwroot\veecar\framework\core\Model.php  
line:34  
function:__construct  
class:Ivy\core\Model  
type:->  
args:array (
  0 => NULL,
)  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:model  
class:Ivy\core\Model  
type:::  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Model.php on line 22

2014/12/03 15:35:03 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：主键异常！  
file:D:\wwwroot\veecar\framework\core\Model.php  
line:34  
function:__construct  
class:Ivy\core\Model  
type:->  
args:array (
  0 => NULL,
)  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:model  
class:Ivy\core\Model  
type:::  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Model.php on line 22

2014/12/03 15:35:03 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：主键异常！  
file:D:\wwwroot\veecar\framework\core\Model.php  
line:34  
function:__construct  
class:Ivy\core\Model  
type:->  
args:array (
  0 => NULL,
)  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:model  
class:Ivy\core\Model  
type:::  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Model.php on line 22

2014/12/03 15:37:48 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：主键异常！  
file:D:\wwwroot\veecar\framework\core\Model.php  
line:34  
function:__construct  
class:Ivy\core\Model  
type:->  
args:array (
  0 => NULL,
)  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:model  
class:Ivy\core\Model  
type:::  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Model.php on line 22

2014/12/03 15:37:51 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：主键异常！  
file:D:\wwwroot\veecar\framework\core\Model.php  
line:34  
function:__construct  
class:Ivy\core\Model  
type:->  
args:array (
  0 => NULL,
)  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:model  
class:Ivy\core\Model  
type:::  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Model.php on line 22

2014/12/03 15:38:58 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：主键异常！  
file:D:\wwwroot\veecar\framework\core\Model.php  
line:34  
function:__construct  
class:Ivy\core\Model  
type:->  
args:array (
  0 => NULL,
)  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:model  
class:Ivy\core\Model  
type:::  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Model.php on line 22

2014/12/03 15:39:01 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：主键异常！  
file:D:\wwwroot\veecar\framework\core\Model.php  
line:34  
function:__construct  
class:Ivy\core\Model  
type:->  
args:array (
  0 => NULL,
)  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:model  
class:Ivy\core\Model  
type:::  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Model.php on line 22

2014/12/03 15:47:12 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：主键异常！  
file:D:\wwwroot\veecar\framework\core\Model.php  
line:34  
function:__construct  
class:Ivy\core\Model  
type:->  
args:array (
  0 => NULL,
)  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:model  
class:Ivy\core\Model  
type:::  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Model.php on line 22

2014/12/03 15:47:36 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：default-分组不存在！  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:129  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'module' => 'default',
      'controller' => 'index',
      'action' => 'index',
    ),
  )),
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:92  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'module' => 'default',
      'controller' => 'index',
      'action' => 'index',
    ),
  )),
  1 => 
  array (
    'name' => '我被覆盖了',
  ),
)  
file:D:\wwwroot\veecar\protected\views\index\view.phtml  
line:9  
function:widget  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 'default/index',
  1 => 
  array (
    'name' => '我被覆盖了',
  ),
)  
file:D:\wwwroot\veecar\framework\core\Template.php  
line:26  
args:array (
  0 => 'D:\\wwwroot\\veecar\\protected\\views\\index\\view.phtml',
)  
function:include_once  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:display  
class:Ivy\core\Template  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Application.php on line 138

2014/12/03 15:47:45 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：default-分组不存在！  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:129  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'module' => 'default',
      'controller' => 'index',
      'action' => 'index',
    ),
  )),
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:92  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'module' => 'default',
      'controller' => 'index',
      'action' => 'index',
    ),
  )),
  1 => 
  array (
    'name' => '我被覆盖了',
  ),
)  
file:D:\wwwroot\veecar\protected\views\index\view.phtml  
line:9  
function:widget  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 'default/index',
  1 => 
  array (
    'name' => '我被覆盖了',
  ),
)  
file:D:\wwwroot\veecar\framework\core\Template.php  
line:26  
args:array (
  0 => 'D:\\wwwroot\\veecar\\protected\\views\\index\\view.phtml',
)  
function:include_once  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:display  
class:Ivy\core\Template  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Application.php on line 138

2014/12/03 15:49:27 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：主键异常！  
file:D:\wwwroot\veecar\framework\core\Model.php  
line:34  
function:__construct  
class:Ivy\core\Model  
type:->  
args:array (
  0 => NULL,
)  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:model  
class:Ivy\core\Model  
type:::  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Model.php on line 22

2014/12/03 15:49:36 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：default-分组不存在！  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:129  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'module' => 'default',
      'controller' => 'index',
      'action' => 'index',
    ),
  )),
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:92  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'module' => 'default',
      'controller' => 'index',
      'action' => 'index',
    ),
  )),
  1 => 
  array (
    'name' => '我被覆盖了',
  ),
)  
file:D:\wwwroot\veecar\protected\views\index\view.phtml  
line:9  
function:widget  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 'default/index',
  1 => 
  array (
    'name' => '我被覆盖了',
  ),
)  
file:D:\wwwroot\veecar\framework\core\Template.php  
line:26  
args:array (
  0 => 'D:\\wwwroot\\veecar\\protected\\views\\index\\view.phtml',
)  
function:include_once  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:display  
class:Ivy\core\Template  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Application.php on line 138

2014/12/03 15:52:25 [error] [application] 
 <!--Fatal error-->:  未捕获的异常Ivy\core\CException  
异常消息：default-分组不存在！  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:129  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'module' => 'default',
      'controller' => 'index',
      'action' => 'index',
    ),
  )),
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:92  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'module' => 'default',
      'controller' => 'index',
      'action' => 'index',
    ),
  )),
  1 => 
  array (
    'name' => '我被覆盖了',
  ),
)  
file:D:\wwwroot\veecar\protected\views\index\view.phtml  
line:9  
function:widget  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 'default/index',
  1 => 
  array (
    'name' => '我被覆盖了',
  ),
)  
file:D:\wwwroot\veecar\framework\core\Template.php  
line:26  
args:array (
  0 => 'D:\\wwwroot\\veecar\\protected\\views\\index\\view.phtml',
)  
function:include_once  
file:D:\wwwroot\veecar\protected\controllers\IndexController.php  
line:42  
function:display  
class:Ivy\core\Template  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:147  
function:viewAction  
class:IndexController  
type:->  
args:array (
)  
file:D:\wwwroot\veecar\framework\core\Application.php  
line:103  
function:dispatch  
class:Ivy\core\Application  
type:->  
args:array (
  0 => 
  Ivy\core\Route::__set_state(array(
     'route' => 
    array (
      'controller' => 'index',
      'action' => 'view',
    ),
  )),
)  
file:D:\wwwroot\veecar\index.php  
line:7  
function:run  
class:Ivy\core\Application  
type:->  
args:array (
)  
异常抛出点：D:\wwwroot\veecar\framework\core\Application.php on line 138

2014/12/04 16:51:07 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:51:07 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:51:08 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:51:08 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:51:08 [error] [application] 系统错误:Memcache::set() [<a href='memcache.set'>memcache.set</a>]: No servers added to memcache connection->file:D:\wwwroot\veecar\framework\core\Cache.php->line:45
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
in D:\wwwroot\veecar\protected\controllers\IndexController.php (17)
2014/12/04 16:52:07 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于目标计算机积极拒绝，无法连接。
 (10061)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:52:07 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于目标计算机积极拒绝，无法连接。
 (10061)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:52:08 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:52:08 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:52:08 [error] [application] 系统错误:Memcache::set() [<a href='memcache.set'>memcache.set</a>]: No servers added to memcache connection->file:D:\wwwroot\veecar\framework\core\Cache.php->line:45
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
in D:\wwwroot\veecar\protected\controllers\IndexController.php (17)
2014/12/04 16:55:17 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:55:17 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:55:18 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于目标计算机积极拒绝，无法连接。
 (10061)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:55:18 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于目标计算机积极拒绝，无法连接。
 (10061)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:55:18 [error] [application] 系统错误:Memcache::set() [<a href='memcache.set'>memcache.set</a>]: No servers added to memcache connection->file:D:\wwwroot\veecar\framework\core\Cache.php->line:45
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
in D:\wwwroot\veecar\protected\controllers\IndexController.php (17)
2014/12/04 16:55:20 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:55:20 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:55:21 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于目标计算机积极拒绝，无法连接。
 (10061)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:55:21 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于目标计算机积极拒绝，无法连接。
 (10061)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:55:21 [error] [application] 系统错误:Memcache::set() [<a href='memcache.set'>memcache.set</a>]: No servers added to memcache connection->file:D:\wwwroot\veecar\framework\core\Cache.php->line:45
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
in D:\wwwroot\veecar\protected\controllers\IndexController.php (17)
2014/12/04 16:55:22 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于目标计算机积极拒绝，无法连接。
 (10061)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:55:22 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于目标计算机积极拒绝，无法连接。
 (10061)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:55:23 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:55:23 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:55:23 [error] [application] 系统错误:Memcache::set() [<a href='memcache.set'>memcache.set</a>]: No servers added to memcache connection->file:D:\wwwroot\veecar\framework\core\Cache.php->line:45
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
in D:\wwwroot\veecar\protected\controllers\IndexController.php (17)
2014/12/04 16:57:40 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:57:40 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:57:41 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:57:41 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:57:41 [error] [application] 系统错误:Memcache::set() [<a href='memcache.set'>memcache.set</a>]: No servers added to memcache connection->file:D:\wwwroot\veecar\framework\core\Cache.php->line:45
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
in D:\wwwroot\veecar\protected\controllers\IndexController.php (17)
2014/12/04 16:57:49 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:57:49 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:57:50 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:57:50 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:57:50 [error] [application] 系统错误:Memcache::set() [<a href='memcache.set'>memcache.set</a>]: No servers added to memcache connection->file:D:\wwwroot\veecar\framework\core\Cache.php->line:45
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
in D:\wwwroot\veecar\protected\controllers\IndexController.php (17)
2014/12/04 16:58:06 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:58:06 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:32
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (32)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:58:07 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Server 127.0.0.1 (tcp 11211) failed with: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:58:07 [error] [application] 系统错误:Memcache::connect() [<a href='memcache.connect'>memcache.connect</a>]: Can't connect to 127.0.0.1:11211, 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
 (10060)->file:D:\wwwroot\veecar\framework\core\Cache.php->line:33
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (33)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
2014/12/04 16:58:07 [error] [application] 系统错误:Memcache::set() [<a href='memcache.set'>memcache.set</a>]: No servers added to memcache connection->file:D:\wwwroot\veecar\framework\core\Cache.php->line:45
in D:\wwwroot\veecar\framework\core\CException.php (41)
in D:\wwwroot\veecar\framework\core\Cache.php (45)
in D:\wwwroot\veecar\protected\controllers\IndexController.php (17)
